sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Deprecated. We’re retiring the Azure Video Analyzer preview service, you're ad
 
 ## 1.0.0b3 (2021-11-09)
 
-- Added device discovery and device detail request for ONVIF enabled devices.
+- Added device discovery and device detail request for ONVIF enabled devices. <!-- cspell: disable-line -->
```

Comment:
Hi @syso-jxx, thanks so much for your PR! Would you mind moving these inline cspell ignores to the cspell.json under a filtered path? We try our best to keep cspell statements out of the source code/docs if we can.

Reply:
Thanks for your review! I moved these inline cspell ignores to the cspell.json under a filtered path.